ideas/168-paid-master-nodes/README.md

```diff
@@ -85,7 +115,9 @@ whitelisting this user's enode for some period of time.
 
 How re-newal will happen is not specified yet.
 
-> Comment from review: May be out of scope for this idea but perhaps we'll want to experiment with different payment models for time - e.g. Alice pays until she runs out of SNT OR N offline messages OR set period of time.
+> Comment from review: May be out of scope for this idea but perhaps we'll want
+> to experiment with different payment models for time - e.g. Alice pays until
+> she runs out of SNT OR N offline messages OR set period of time.
 
 Solving the initial user stories don't require any proof of message delivery or
 quality of service. Reputation of specific mail servers thus happens completely
```

Comment:
No idea what `Reputation of specific mail servers thus happens completely` means?

Reply:
Does this make it clearer? https://github.com/status-im/ideas/pull/216/commits/b02330365e613860396411e0841d31142242e507